pdns/recursordist/docs/lua-scripting/hooks.rst

```diff
@@ -197,7 +197,10 @@ Interception Functions
   - :attr:`dq.localaddr <DNSQuestion.localaddr>`
   - :attr:`dq.qname <DNSQuestion.qname>`
   - :attr:`dq.qtype <DNSQuestion.qtype>`
-  - :attr:`dq.isTcp <DNSQuestion.isTcp>`
+  - :attr:`dq.isTcp <DNSQuestion.isTcp>` this attribute may be changed by the hook to force the use of UDP or TCP
```

Comment:
Also add a 
```
.. versionchanged:: 5.3.0
   
  `isTcp` can be set to force TCP or UDP
```
```suggestion
  - :attr:`dq.isTcp <DNSQuestion.isTcp>` since version 5.3.0 this attribute may be changed by the hook to force the use of UDP or TCP
```

Reply:
also change `The DNS question to handle` to `Attributes of a DNS question to be sent out to the an authoritative server`.